build.gradle

```diff
@@ -2,17 +2,19 @@
 
 buildscript {
     repositories {
+        google()
```

Comment:
I'd prefer not to use features from 3.0 until it hits stable, let's revert this for now.